MA-Bylaws.md

```diff
@@ -448,6 +448,29 @@ member by the MA.
 
 For clarification: other projects members need not be MA members.
 
+A MAP member can become an MA member according to § 3.
+
+MA has the right to revoke the registration of a MAP if (a) MA bylaws
+are violated, (b) laws or regulations that MA has to follow are
+violated, or (c) the MAP is changed so that it is no longer compliant
+with MA.
+
+If the MA Board suspects that one of the situations from the previous
+paragraph occurred it can within 2 months request the MA Project in
+writing (including email) to remedy the situation. If the project does
+not remove the cause within 2 months after the request of the MA Board,
+then the project can be revoked from the MA. For this exclusion a
+qualified majority of an MA Assembly (see § 14) is necessary.
+
+Resignation of a project as a MAP applies immediately after MA received
+the written notice.
+
+The MA can conditionally accept a MAP with *probationary status* to allow the
```

Comment:
I like the idea of the probation status. This should encourage e.g. publicly funded research proejcts to align with other MAPs early and not after their end.
However, it is not yet clearly defined, what "probation status" means.
What are the "rights" and "duties" of the "MAP in probation" and the MA during that period?
Can an MA Project in "probation" status release standards? Or only prereleases? Can the project leader become MA board member? ...

Reply:
Even full MA projects cannot release standards, right? The MA votes on those today (after the MAP approved it).

Reply:
> 
> 
> I like the idea of the probation status. This should encourage e.g. publicly funded research proejcts to align with other MAPs early and not after their end.
> However, it is not yet clearly defined, what "probation status" means.
> What are the "rights" and "duties" of the "MAP in probation" and the MA during that period?
> Can an MA Project in "probation" status release standards? Or only prereleases? Can the project leader become MA board member? ...

The text does not specify that intentionally - therefore, there is no difference and therefore no excuse to be or become like all other MAPs.

The most important role of the vote is to require a discussion about if a MAP on probation really belongs into the MA and in case it is a clear miss, to start de-registration (to protect the scares MA resources and the keep the brand of MA standards strong).
This will hopefully make it easier for us to accept fledgling projects and nurture them and give "old" MAPs time to align to the needs of the new MAP as well.

Reply:
> Even full MA projects cannot release standards, right? The MA votes on those today (after the MAP approved it).

The Bylaws say:
`MAPs are encouraged to apply for approval of their results by MA as MA
standard according to §2.`

I would read this as if they can standardize as they like, but that will only be an MA standard, once approved by the MA.

Reply:
I want to see this in the light of the MMI project, that really wanted to release their standard this year, clearly without the time needed to first make sure it is a standard that is coordinated with existing standards and not competing with existing standards.  Since the MA standards are supposed to be coordinated, and that this – as long as it is true – adds a huge amount of value to each of the coordinated standards, I am strongly against their current specification being released as an MA standard.

Yes, depending on interpretation of the bylaws it might be technically possible for the MA members to reject the release of an un-coordinated standard, but:
- I think we would have sent the wrong signals if we even give them a chance to release their work to the MA voting; it could come as a very unpleasant backlash for the MAP if their standard gets rejected by the MA.
- There is a strong tradition of not stopping the standards proposed to the MA by the MAPs, and I am afraid that too many members that are not initiated in the details would vote in favor out of habit or expectation.

If they want to work towards a coordinated standard, I welcome them to do so under the MA umbrella, and for this we'd need to give their project something like a probationary status.  I'd only feel comfortable giving them this kind of membership provided that it is made clear from the beginning that their current specification is not ready to become an MA standard, and a very clear way to say this is that a probationary MAP cannot release MA standards.  If this means that they are not interested, so be it, but if I were them I'd release what I have first as a non-MA standard, and then apply to become a probationary MAP with the intention of developing the specification to the point that it looks like a well coordinated standard and the probationary status can be removed.

Reply:
By the way, if a probationary MAP cannot release MA standards, I don't see a compelling need to limit the time a MAP can have probationary status.  When the MAP feels ready, they should apply for full MAP status, and this may take shorter or longer depending on how far away they start from a realistic MA standard, and of course how actively they work on getting there.